packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,64 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorBy,
+    WaferMapColorsScale,
+    WaferMapDataType,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'wafer-quadrant'
+    })
+    public waferQuadrant!: WaferMapQuadrant;
+
+    @attr({
+        attribute: 'wafer-orientation'
+    })
+    public waferOrientation!: WaferMapOrientation;
+
+    @attr({
+        attribute: 'max-characters'
+    })
+    public maxCharacters!: number;
+
+    @attr({
+        attribute: 'wafer-datatype'
+    })
+    public waferDataType!: WaferMapDataType;
+
+    @attr({
+        attribute: 'colorBy'
+    })
+    public colorBy: WaferMapColorBy = WaferMapColorBy.hardBin;
+
+    @attr({
+        attribute: 'highlighted-values'
+    })
+    public highlightedValues!: string[];
+
+    public dice!: WaferMapDie[];
```

Comment:
I would probably consider making this an observable as well - if we want to update the rendered wafermap in case there is a data change. However, if we do not expect data change after the initial render, this could stay as a simple property.

Reply:
Added :)